sound/soc/intel/common/soc-acpi-intel-lnl-match.c

```diff
@@ -240,6 +286,12 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_lnl_sdw_machines[] = {
 		.drv_name = "sof_sdw",
 		.sof_tplg_filename = "sof-lnl-rt722-l0.tplg",
 	},
+	{
+		.link_mask = GENMASK(2, 0),
+		.links = lnl_sdw_rt1318_l12_rt714_l0,
+		.drv_name = "sof_sdw",
+		.sof_tplg_filename = "sof-lnl-rt1318-l12-rt714-l0.tplg"
```

Comment:
out of curiosity, do we have such a topology at all? I can see `sof-mtl-rt1318-l12-rt714-l0` but no LNL variant.

Reply:
@ujfalusi , it needs new topology on LNL and creates here https://github.com/thesofproject/sof/pull/9066